app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -18,6 +18,9 @@
 use Mautic\CoreBundle\Helper\EmojiHelper;
 use Mautic\LeadBundle\Entity\DoNotContact;
 use Mautic\LeadBundle\Entity\Lead;
+use Mautic\LeadBundle\Entity\StatDevice;
+use Mautic\LeadBundle\Form\Type\TagListType;
+use Mautic\LeadBundle\LeadEvents;
```

Comment:
This is probably because of a bad merge but it's throwing a fatal error because it's already defined in line 25.